src/main/java/org/gridsuite/modification/server/entities/equipment/creation/BranchCreationEntity.java

```diff
@@ -45,15 +45,15 @@ public class BranchCreationEntity extends EquipmentCreationEntity {
     @Column(name = "busOrBusbarSectionId2")
     private String busOrBusbarSectionId2;
 
-    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER)
```

Comment:
the thing is, if I don't switch it to eager when I try to copy an item with a currentLimit i get an exception uninitialized proxy passed to proxy. I didn't find a better way to avoid the exception, any suggestion ? this might be true also for other entities with with relationship (need testing)

Reply:
went with custom request to fix this issue